src/strategies/eth-wallet-age/index.ts

```diff
@@ -0,0 +1,71 @@
+import { EnumType } from 'json-to-graphql-query';
+import fetch from 'cross-fetch';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'chaituvr';
+export const version = '0.1.0';
+
+const getJWT = async (dfuseApiKey) => {
+  const rawResponse = await fetch('https://auth.dfuse.io/v1/auth/issue', {
+    method: 'POST',
+    headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({"api_key": dfuseApiKey})
+  });
+  const content = await rawResponse.json();
+  return content.token
+};
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+  ) {
+  const apiLimit = 300;
+  let data: any =  []
+  const query = Object.fromEntries(addresses.map(address => [`_${address}`, {
+    __aliasFor: 'searchTransactions',
+    __args: {
+      indexName: new EnumType('CALLS'),
+      query: `(from:${address} OR to:${address})`,
+      sort: new EnumType('ASC'),
+      limit: 1,
+    },
+    edges: {
+      block: {
+        header: {
+          timestamp: true
+        },
+        number: true
+      },
+      node: {
+        from: true,
+        to: true
+      }
+    }
+  }]));
+  
+  const dfuseJWT = await getJWT(options.dfuseApiKey || 'web_f527db575a38dd11c5b686d7da54d371');
+  data = await subgraphRequest('https://mainnet.eth.dfuse.io/graphql', query, {
```

Comment:
Is this available on others network? Rinkeby, Kovan etc? If yes we may want to handle it with something like this: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/balancer/index.ts#L7-L10

Reply:
Only available for Mainnet and Ropsten... but Ropsten isn't available in https://github.com/snapshot-labs/snapshot.js/blob/master/src/networks.json  🤔